CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## 1.4.0
+
+This is a feature and maintenance release.
+
+* [TK-247](https://tickets.puppetlabs.com/TK-247) Added tests for Path
+  Traversal Attacks.
+* Add `add-websockets-handler` to WebserverService and WebroutingService
```

Comment:
It looks like we didn't.  Can you add something to the README that says somethign along the lines of:
```
NOTE: experimental feature; the API for websockets support may be subject to minor changes in a future release.
```